app/models/manageiq/providers/vmware/network_manager/refresh_parser.rb

```diff
@@ -28,36 +28,90 @@ def ems_inv_to_hashes
 
     private
 
-    def get_networks
-      # fetch org VDC networks
-      @inv[:networks] = @org.networks
+    def log_header
+      "MIQ(#{self.class.name}.#{__method__}) Collecting data for EMS name: [#{@ems.name}] id: [#{@ems.id}]"
```

Comment:
`log_header` function must not include the part for `Collecting data for ...` -- this is only useful in `ems_inv_to_hashes` where you are showing the high level description. All other log messages should only contain information about the location of the message.

BTW, I haven't tested, but will `__method__` return the name of the actual function or simply `log_header` function?

Reply:
It'll return `"log_header"`, if you want to do this you'll need to use `caller_location` like we do in [`vmdb_logger`](https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/logging.rb#L30-L35)